### PR TITLE
Initialize new testnets with updated values

### PR DESCRIPTION
**Pull request by measure-fi**
[Public Testnet EPIC](https://www.notion.so/volumefi/EPIC-06-17-Paloma-Public-Testnest-1-d11ce23143c7475a9157acd224933d81)

# Background

We want a number of different values for testnet initialization, add them here.

# Testing completed

- [x] Test initializing a new chain

### CODE REVIEW

File: scripts/setup-volume-testnet.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name="$1"
+  address="$2"
+  
+  palomad add-genesis-account "$validator" 250000000grain
+  palomad gentx "$name" "$VALIDATOR_STAKE_AMOUNT" --chain-id "$CHAIN_ID"
+}
+
+init_validator chase paloma1x6m0d47lcdka7mrg7997u80cyq60yg5adeekvx
+init_validator jason paloma1x6m0d47lcdka7mrg7997u80cyq60yg5adeekvx
+init_validator taariq paloma1nvhryyg63e8vw9t5sx9ht8l77jzgjvrq6jcdnp
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:33]: @taariq What address should be used here (currently just giving Vera double)

File: scripts/setup-volume-testnet.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+init_validator() {
+  name="$1"
+  address="$2"
+  
+  palomad add-genesis-account "$validator" 250000000grain
+  palomad gentx "$name" "$VALIDATOR_STAKE_AMOUNT" --chain-id "$CHAIN_ID"
+}
+
+init_validator chase paloma1x6m0d47lcdka7mrg7997u80cyq60yg5adeekvx
+init_validator jason paloma1x6m0d47lcdka7mrg7997u80cyq60yg5adeekvx
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:32]: @rootedbox What address should we use here (currently just giving me double)?

File: scripts/setup-volume-testnet.sh [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+init_validator() {
+  name="$1"
+  address="$2"
+  
+  palomad add-genesis-account "$address" 250000000grain
+  palomad gentx "$name" "$VALIDATOR_STAKE_AMOUNT" --chain-id "$CHAIN_ID"
+}
+
+# Faucet
+palomad add-genesis-account paloma1fgy8ghc70swdzzpa4xpk544xs7q8xh3zfkzal2 4000000000grain
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:32]: Do we want less in the faucet and any excess (up to 5B) stored in a 6th account?

File: scripts/setup-volume-testnet.sh [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+jq 'walk(if type == "string" and .. == "stake" then "grain" else . end)' genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+popd
+
+VALIDATOR_STAKE_AMOUNT=100000000grain
+init_validator() {
+  name="$1"
+  address="$2"
+  
+  palomad add-genesis-account "$address" 250000000grain
+  palomad gentx "$name" "$VALIDATOR_STAKE_AMOUNT" --chain-id "$CHAIN_ID"
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:28]: @Vizualni Currently getting `Error: failed to fetch 'chase' from the keyring: chase.info: key not found` from this line, which makes sense. How do I add external keys without importing the private key?

Comment (measure-fi) [reply scripts/setup-volume-testnet.sh:28]: This makes sense. I think the "chase" validator will have to execute this line and I'll make a new one every script invocation.

File: scripts/setup-volume-testnet.sh [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+sed -i 's/^laddr = ".*:26657"/laddr = "tcp:\/\/0.0.0.0:26657"/' config.toml
+jq ".chain_id = \"${CHAIN_ID}\"" genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+jq 'walk(if type == "string" and .. == "stake" then "grain" else . end)' genesis.json > genesis.json.tmp && mv genesis.json.tmp genesis.json
+popd
+
+name="chase"
+echo "$MNEMONIC" | palomad keys add "$name" --recover
+address="$(palomad keys show "$name" -a)"
+
+palomad add-genesis-account "$address" 4000000000grain
```
Comment (measure-fi) [inline scripts/setup-volume-testnet.sh:27]: @verabehr As written this account has 4Ggr, the below accounts get 800Mgr total. What would you like the distribution to be?

Comment (verabehr) [reply scripts/setup-volume-testnet.sh:27]: ok, so let's change the faucet to give out 100 000grain per request. 
us for (jason, taariq, you, me) each get 250000000 as you have it. 
the faucet starts out with 500 000 000grain as you have it
that gives us 1 500 000 000grain in total.

Comment (measure-fi) [reply scripts/setup-volume-testnet.sh]: Done.

**Pull request merged**